style.css

```diff
@@ -681,6 +701,11 @@ h6 {
 	font-size: 18px;
 }
 
+strong,
+b {
+	font-weight: 400;
```

Comment:
@philcable why set `strong` and `b` as the same weight as the paragraph text?

Reply:
Good point! I just blindly copied that over from Edit CSS - I'll bump it to 600.